### ISSUE TITLE
Global distribution view/graph

### ISSUE THREAD

**Issue by woj-tek** [mtotschnig/MyExpenses#405]
It would be nice to have Global Distribution view (so for example if someone is using credit card and debit card attached to different accounts, and it's get more complicated when using more than two) so it would be easier to track spending vs income. I'm aware, that this is most likely blocked by #362 (and fully understand why - in my case I'm using 3 currencies at daily basis, *sigh*...)

**Comment by mtotschnig** [comment]
I am not sure I understand, what exactly is your expectation from this ticket? Do you know that distribution for all accounts that share the same currency is already available? #362 implies that once it is possible to convert between currencies, a global distribution graph will be available.

**Comment by woj-tek** [comment]
mea culpa - indeed one can select currency and there is a more general overview. However in my case it's still a bit less useful due to lack of #362
